lib/elixir/unicode/SpecialCasing.txt

```diff
@@ -66,7 +66,7 @@
 
 # Preserve canonical equivalence for I with dot. Turkic is handled below.
 
-0130; 0069 0307; 0130; 0130; # LATIN CAPITAL LETTER I WITH DOT ABOVE
+0130; 0069; 0130; 0130; # LATIN CAPITAL LETTER I WITH DOT ABOVE
```

Comment:
Those files are generated by Unicode and we should not modify them. I believe you may need to add this value to `@conditional_downcase` in properties.ex. Thanks!

Reply:
Ok..
Because currently, doing `String.downcase("İ") == "i"` returns `false`.
In fact I noticed that, when the test failed.

Reply:
Yes, we want to fix that, but only for `:turkic`. :)

Reply:
Regarding the @condtional_downcase mapping (currently having greek sigma), I thought it's about mapping that are conditional within the language, while in Turkic, there aren't conditional, they apply everytime...

Reply:
Oh..
I got it..
It's in order to bypass the default generation..

Reply:
@altdsoy I remember now! The reason why we do it with `@conditional_mapping` is because we didn't want to match on `mode == :greek` because it may affect performance. We just want to check the mode once we are inside the character that has a special case and the `@conditional_downcase` then avoids a warning. For now, leave this particular part of the code as is, I can refactor and run benchmarks later. :)